src/app/tide/compiler.h

```diff
@@ -186,13 +161,13 @@ class TheoreticalPeakCompiler {
     pos_ += 4;
   }
 
-  void AddNegative(int peak) {
+/*  void AddNegative(int peak) {
```

Comment:
Why not delete this? Is there a reason why you commented it out instead?

Reply:
Well, I know I should, but this assembly code is just beyond my knowledge, so I decided to keep it in front in case we need to put it back. I would not be able to reproduce/reimplement that code. If you insist, I'll delete that code. Let me know.

Reply:
Ok, I understand the rationale. No need to delete it.